src/web/utils/FocusManager.ts

```diff
@@ -114,7 +114,7 @@ export class FocusManager extends FocusManagerBase {
                 storedComponent.limitedCount === 0 &&
                 storedComponent.limitedCountAccessible === 0)
             .map(storedComponent => ReactDOM.findDOMNode(storedComponent.component) as HTMLElement)
-            .filter(el => el && el.focus);
+            .filter(el => el && el.focus && (el.tabIndex !== -1));
```

Comment:
If I'm reading the HTML spec correctly, we should be checking for any negative number here, not just -1.

Reply:
Yes, that's correct, we just never use anything lower than -1. Updated to consider any negative tabIndex.